portfolio/src/main/webapp/script.js

```diff
@@ -46,6 +50,25 @@ function showMap() {
   const map = new google.maps.Map(
       document.getElementById('map'),
       {center: DEFAULT_CENTER, zoom: DEFAULT_ZOOM});
+  
+  addMarkerInfo(map, LIBERTY_LOCATION, "The old theater I used to work at!"); 
+  addMarkerInfo(map, HS_LOCATION, "My old high school.");
+  addMarkerInfo(map, FOOD_LOCATION, "My favorite restaurant.");     
```

Comment:
This is fine as we don't have a lot of data here, another way to do this is:
1. make a constant map with coordinates and info.
2. Write a for loop to access the map and call addMarkerInfo in the for loop.